editor.planx.uk/src/@planx/components/Pay/Public/Confirm.tsx

```diff
@@ -81,19 +81,25 @@ const PayBody: React.FC<PayBodyProps> = (props) => (
             size="large"
             onClick={props.onConfirm}
           >
-            {props.buttonTitle || "Pay using GOV.UK Pay"}
+            {props.buttonTitle || "Pay now using GOV.UK Pay"}
           </Button>
           {props.showInviteToPay && (
             <>
-              <Typography variant="body2">or</Typography>
-              <Link
-                component="button"
+              <Button
+                variant="contained"
+                color="secondary"
+                style={{ borderBottom: `solid 2px lightgrey` }}
+                size="large"
                 onClick={props.changePage}
                 disabled={Boolean(props?.paymentStatus)}
                 data-testid="invite-page-link"
               >
+                {"Invite someone else to pay for this application"}
+              </Button>
+              <Typography variant="body2">or</Typography>
+              <Link component="button">
                 <Typography variant="body2">
-                  Invite someone else to pay for this application
+                  Save and return to this application later
```

Comment:
this doesn't actually do anything `onClick` yet! it's flagged as an issue in the checklist on this Trello card & behind the feature flag: https://trello.com/c/45whNb0u/2385-99-outstanding-itp-tasks

hardcoded here so we have visual consistency in show & tell, but should be able to remove this `<Link>` entirely on both pages once properly addressed.